README.md

```diff
@@ -10,8 +10,10 @@ For this example we are using a remote control LEGO Truck (Mercedes-Benz Arocs)
 ## Required hardware
```

Comment:
It's not in this pull request but ... Streamming -> Streaming

Reply:
I will check for typos again, thanks for the heads up